resources/l10n/ru/LC_MESSAGES/loot.po

```diff
@@ -1,50 +1,285 @@
+#: src/gui/cef/query/query_executor.h:43
+msgid ""
+"Oh no, something went wrong! You can check your LOOTDebugLog.txt (you can "
+"get to it through the main menu) for more information."
+msgstr ""
+"О нет, что-то пошло не так! Дополнительную информацию можно узнать в файле "
+"LOOTDebugLog.txt (можно получить доступ к нему через главное меню)."
+
+#: src/gui/cef/query/types/get_game_data_query.h:46
+msgid "Parsing, merging and evaluating metadata..."
+msgstr "Обработка, объединение и сверка с метаданными.."
+
+#: src/gui/cef/query/types/metadata_query.h:201
+#: src/gui/cef/query/types/metadata_query.h:228
+msgid "\"%1%\" contains a condition that could not be evaluated. Details: %2%"
+msgstr "\"%1%\" включает выражение, которое не может быть оценено. Детали: %2%"
+
+#: src/gui/cef/query/types/metadata_query.h:250
+#: src/gui/cef/query/types/metadata_query.h:251
+msgid "N/A: No masterlist present"
+msgstr "N/A: Мастерлист отсутствует"
+
+#: src/gui/cef/query/types/metadata_query.h:257
+#: src/gui/cef/query/types/metadata_query.h:258
+msgid "Unknown: Git repository missing"
+msgstr "Неизвестно: Отсутствует хранилище Git"
+
+#: src/gui/cef/query/types/metadata_query.h:266
+#: src/gui/cef/query/types/metadata_query.h:267
+msgid "(edited)"
+msgstr "(изменено)"
+
+#: src/gui/cef/query/types/sort_plugins_query.h:53
+msgid "Sorting load order..."
+msgstr "Сортировка порядка загрузки..."
+
+#: src/gui/state/loot_state.cpp:95
+msgid "Error: --auto-sort was passed but no --game parameter was provided."
+msgstr "Error: --автосортировка прошла, но --параметр игры не был указан."
+
+#: src/gui/state/loot_state.cpp:118
+msgid "Error: Could not create LOOT settings file. %1%"
+msgstr "Ошибка: Невозможно создать файл настроек LOOT. %1%"
+
+#: src/gui/state/loot_state.cpp:128
+msgid "Error: Settings parsing failed. %1%"
+msgstr "Ошибка: Обработка настроек не удалась. %1%"
+
+#: src/gui/state/loot_state.cpp:191
+msgid "Error: Game-specific settings could not be initialised. %1%"
+msgstr ""
+"Ошибка: Специфические для игры настройки не могут быть инициализированы. %1%"
+
+#: src/gui/state/game/game.cpp:169 src/gui/state/game/game.cpp:199
+msgid "This plugin requires \"%1%\" to be installed, but it is missing."
+msgstr "Этот плагин требует установленного плагина \"%1%\", но он отсутствует."
+
+#: src/gui/state/game/game.cpp:182
+msgid "This plugin requires \"%1%\" to be active, but it is inactive."
+msgstr "Этот плагин требует подключенного плагина \"%1%\", но он не подключен."
+
+#: src/gui/state/game/game.cpp:218
+msgid "This plugin is incompatible with \"%1%\", but both files are present."
+msgstr "Этот плагин несовместим с \"%1%\", но оба файла присутствуют."
+
+#: src/gui/state/game/game.cpp:253
+msgid ""
+"This plugin is a light master and requires the non-master plugin \"%1%\". "
+"This can cause issues in-game, and sorting will fail while this plugin is "
+"installed."
+msgstr ""
+"Этот плагин является легким мастером и требует не-мастер плагин \"% 1%\". "
```

Comment:
Looks like an extra space snuck in here.

Reply:
Yeah. Even dont know where this space from. Usually just coping original and changing only text in translation field.